### PR TITLE
[TE] frontend - harleyjj/alert-details - fix timeseries fetch

### PR DESCRIPTION
**Pull request by harleyjj**
Removes granularity and timezone from timeseries fetch in Alert Overview and Preview

### CODE REVIEW

File: thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js [hunk 1]
```diff
@@ -597,12 +597,10 @@ export default Component.extend({
       analysisRange,
       selectedBaseline
     } = this.getProperties('metricUrn', 'analysisRange', 'selectedBaseline');
-    const granularity = '15_MINUTES';
-    const timezone = moment.tz.guess();
 
     set(this, 'errorTimeseries', null);
 
-    const urlCurrent = `/rootcause/metric/timeseries?urn=${metricUrn}&start=${analysisRange[0]}&end=${analysisRange[1]}&offset=current&granularity=${granularity}&timezone=${timezone}`;
+    const urlCurrent = `/rootcause/metric/timeseries?urn=${metricUrn}&start=${analysisRange[0]}&end=${analysisRange[1]}&offset=current`;
```
Comment (xiaohui-sun) [inline thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js]: Should get the granularity and timezone from detect function.
E.g., the underline data may have hourly granularity, but we may want to monitor it in daily granularity.

Comment (jihaozh) [reply thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js]: The time zone should be the frontend's timezone. 

Technically, the granularity of the graph should be the granularity for detection. However, with yaml, we allow the user to monitor in different granularity in different detection rules. Therefore, multiple granularities could be used in the in one detection, we don't know which one to show. I suggest using default time granularity for the metric in preview graph for now.

File: thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js [hunk 2]
```diff
@@ -597,12 +597,10 @@ export default Component.extend({
       analysisRange,
       selectedBaseline
     } = this.getProperties('metricUrn', 'analysisRange', 'selectedBaseline');
-    const granularity = '15_MINUTES';
-    const timezone = moment.tz.guess();
```
Comment (jihaozh) [inline thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js]: Should we keep the timezone? @harleyjj could you check how RCA makes this call, it should use the same timezone. 

The granularity can be just left empty.

Comment (jihaozh) [reply thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js]: Just checked this. 

The timezone should be pinned to "America/Los_Angeles".
Granularity should be left empty.

Thanks.

Comment (harleyjj) [reply thirdeye/thirdeye-frontend/app/pods/components/alert-details/component.js]: I'll go ahead and change that asap

**Pull request merged**